src/microbe_stage/CompoundCloudSystem.cs

```diff
@@ -91,16 +91,20 @@ public void Init(FluidSystem fluidSystem)
             clouds.Remove(cloud);
         }
 
+        int renderPriority = -1;
```

Comment:
Do positive values not work or why these are negatives? We already use positive render priority values for organelles so using exact opposite values here will reduce code uniformity.

Reply:
Okay, then this is the right way then. Could you add a comment explaining this here to future programmers?